src/frontend/youtube_clone/src/app/profile-page/profile-page.component.html

```diff
@@ -0,0 +1,3 @@
+<p>profile-page works!</p>
+<button type = button routerLink = '../video-page'> click me for video </button>
+<button type = button routerLink = '../main-page'> click me for main page </button>
```

Comment:
I already saw that code in the `MainPageComponent`!

Reply:
Cleaned up and solved